### PR TITLE
Add group.create action + tests.

### PR DESCRIPTION
**Pull request by reiterl**

### CODE REVIEW

File: openslides_backend/action/group/create.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        },
+    )
+
+    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        """
+        Set default permissions to list().
+        """
+        if instance.get("permissions") is None:
+            instance["permissions"] = []
+        return instance
```
Comment (jsangmeister) [inline openslides_backend/action/group/create.py:29]: This should not be necessary, the "default default" for array fields is already `[]`. Please remove and test if it still works. Independent from that, please implement defaults that differ from the "default defaults" in the models.yml and not in the actions themselves to keep the handling consistent.

File: openslides_backend/action/group/create.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+class GroupCreate(CreateAction):
+    """
+    Action to create a group.
+    """
+
+    model = Group()
+    schema = DefaultSchema(Group()).get_create_schema(
+        required_properties=["name", "meeting_id"],
+        additional_optional_fields={
+            "permissions": {"type": "array", "items": {"type": "string"}}
```
Comment (jsangmeister) [inline openslides_backend/action/group/create.py:19]: `permissions` is a valid field in the group model, can't you just pass it in the `optional_properties`?

File: tests/system/action/group/test_create.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
+from tests.system.action.base import BaseActionTestCase
+
+
+class UserCreateActionTest(BaseActionTestCase):
```
Comment (jsangmeister) [inline tests/system/action/group/test_create.py:4]: Please rename

File: tests/system/action/group/test_create.py [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+                {
+                    "action": "group.create",
+                    "data": [{"name": "test_Xcdfgee", "meeting_id": 22}],
+                }
+            ],
+        )
+        self.assert_status_code(response, 200)
+        model = self.get_model("group/1")
+        assert model.get("name") == "test_Xcdfgee"
+        assert model.get("meeting_id") == 22
```
Comment (jsangmeister) [inline tests/system/action/group/test_create.py:19]: assert that `permissions == []`

File: tests/system/action/group/test_create.py [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+                            "permissions": ["test"],
+                        }
+                    ],
+                }
+            ],
+        )
+        self.assert_status_code(response, 200)
+        model = self.get_model("group/1")
+        assert model.get("name") == "test_Xcdfgee"
+        assert model.get("meeting_id") == 22
```
Comment (jsangmeister) [inline tests/system/action/group/test_create.py:41]: assert that `permissions == ["test"]`

**Review by jsangmeister — CHANGES_REQUESTED** [review]

File: tests/system/action/group/test_create.py (hunk 5 above)
Comment (reiterl) [reply tests/system/action/group/test_create.py:41]: Done.

File: tests/system/action/group/test_create.py (hunk 4 above)
Comment (reiterl) [reply tests/system/action/group/test_create.py:19]: Done.

File: tests/system/action/group/test_create.py (hunk 3 above)
Comment (reiterl) [reply tests/system/action/group/test_create.py:4]: Renamed in GroupCreateActionTest.

File: openslides_backend/action/group/create.py (hunk 2 above)
Comment (reiterl) [reply openslides_backend/action/group/create.py:19]: Yes, it can, changed.

File: openslides_backend/action/group/create.py (hunk 1 above)
Comment (reiterl) [reply openslides_backend/action/group/create.py:29]: Okay, cutted and tested, okay.

**Review by jsangmeister — APPROVED** [review]
Please sqash

**Pull request merged**